pkg/controller/atlasbackupcompliancepolicy/atlasbackupcompliancepolicy_controller_test.go

```diff
@@ -0,0 +1,354 @@
+package atlasbackupcompliancepolicy
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap/zaptest"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	atlasmock "github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/mocks/atlas"
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/customresource"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/workflow"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/indexer"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestReconcile(t *testing.T) {
```

Comment:
All these tests do a reconcile and result verification, I best most if not all could be tabulated and would save code and easy readability. Still just a nitpick, not a requirement.

Reply:
me and @roothorp paired on this and made it a table driven test. @roothorp will do the same for `TestFindBCPForProjects`.